CIScripts/SpawnVM.ps1

```diff
@@ -0,0 +1,201 @@
+        })
+
+        return $Sessions
+    }
+
+    function Enable-NBLDebugging {
+        Param ([Parameter(Mandatory = $true)] [System.Management.Automation.Runspaces.PSSession] $Session)
+
+        Invoke-Command -Session $Session -ScriptBlock {
+            New-ItemProperty -Path "HKLM:\System\CurrentControlSet\Services\NDIS\Parameters" -Name TrackNblOwner -Value 1 -PropertyType DWORD -Force | Out-Null
+        }
+    }
+
+    function Initialize-CrashDumpSaving {
+        Param ([Parameter(Mandatory = $true)] [System.Management.Automation.Runspaces.PSSession] $Session,
+               [Parameter(Mandatory = $true)] [string] $DumpFilesLocation,
+               [Parameter(Mandatory = $true)] [string] $DumpFilesBaseName)
+
+        $DumpFilename = $DumpFilesLocation + "\" + $DumpFilesBaseName + "_" + $Session.ComputerName + ".dmp"
+
+        Invoke-Command -Session $Session -ScriptBlock {
+            Get-Disk | Where-Object PartitionStyle -eq 'raw' |
+                Initialize-Disk -PartitionStyle MBR -PassThru |
+                New-Partition -AssignDriveLetter -UseMaximumSize |
+                Format-Volume -FileSystem NTFS -NewFileSystemLabel "DUMP_HD" -Confirm:$false | Out-Null
+
+            New-ItemProperty -Path "HKLM:\SYSTEM\CurrentControlSet\Control\CrashControl" -Name DumpFile -PropertyType ExpandString -Value $Using:DumpFilename -Force | Out-Null
+        }
+    }
+
+    function Install-Artifacts {
+        Param ([Parameter(Mandatory = $true)] [System.Management.Automation.Runspaces.PSSession] $Session,
+               [Parameter(Mandatory = $true)] [string] $ArtifactsDir)
+
+        Push-Location $ArtifactsDir
+
+        Invoke-Command -Session $Session -ScriptBlock {
+            New-Item -ItemType Directory -Force C:\Artifacts | Out-Null
+        }
+
+        Write-Host "Copying Docker driver installer"
+        Copy-Item -ToSession $Session -Path "docker_driver\installer.msi" -Destination C:\Artifacts\
+
+        Write-Host "Copying Contrail vRouter API"
+        Copy-Item -ToSession $Session -Path "agent\contrail-vrouter-api-1.0.tar.gz" -Destination C:\Artifacts\
+
+        Write-Host "Copying vtest"
+        Copy-Item -ToSession $Session -Path "vrouter\utils\vtest" -Destination C:\Artifacts\ -Recurse -Force
+
+        Write-Host "Copying vRouter and Utils MSIs"
+        Copy-Item -ToSession $Session -Path "vrouter\vRouter.msi" -Destination C:\Artifacts\
+        Copy-Item -ToSession $Session -Path "vrouter\utilsMSI.msi" -Destination C:\Artifacts\
+        Copy-Item -ToSession $Session -Path "vrouter\*.cer" -Destination C:\Artifacts\ # TODO: Remove after JW-798
+
+        Invoke-Command -Session $Session -ScriptBlock {
+            Write-Host "Installing Contrail vRouter API"
+            pip2 install C:\Artifacts\contrail-vrouter-api-1.0.tar.gz | Out-Null
+
+            Write-Host "Installing Docker driver"
+            Start-Process msiexec.exe -ArgumentList @("/i", "C:\Artifacts\installer.msi", "/quiet") -Wait
```

Comment:
I know this is not in scope of this PR, but if you could rename installer.msi to something like driver.msi and utilsMSI.msi to something like utils.msi, that would be great